src/bin/errors.ts

```diff
@@ -49,6 +49,12 @@ class ErrorCLIPolykeyAgentProcess<T> extends ErrorCLI<T> {
   exitCode = sysexits.OSERR;
 }
 
+class ErrorCLIPolykeyAsynchronousDeadlock<T> extends ErrorCLI<T> {
```

Comment:
This should not be a CLI error. This should be part of all the other errors like UnhandledPromiseRejection. We could have them in ExitHandlers.

Actually all of these can be in bin/errors.ts. But it needs to be separate from ErrorCLI, also we should change ErrorCLI to ErrorBin.